README.md

```diff
@@ -2144,8 +2144,8 @@ Other Style Guides
 
 ## Blocks
 
-  <a name="blocks--braces"></a><a name="16.1"></a>
-  - [16.1](#blocks--braces) Use braces with all multiline blocks. eslint: [`nonblock-statement-body-position`](https://eslint.org/docs/rules/nonblock-statement-body-position)
+  <a name="blocks--braces"></a>
+  - [16.1](#blocks--braces) Use braces with `if`, `else`, `while`, `do-while`, and `for` statements when writing multiline blocks. eslint: [`nonblock-statement-body-position`](https://eslint.org/docs/rules/nonblock-statement-body-position)
```

Comment:
That's because function bodies aren't a block, they're a function body.

The eslint rule doesn't ever have to cover the entirety of the guide section - the guide is the source of truth, and the guide applies to all blocks.

Reply:
So their is no need to update according to eslint rule.

Reply:
ok, got the point.